### PR TITLE
#46 hotel info page add possibility to upload hotel logo

### PR DESCRIPTION
**Pull request by GojkovicGoran**

### CODE REVIEW

File: con-base/src/main/java/ba/codecentric/base/domain/Hotel.java [hunk 1]
```diff
@@ -75,6 +76,10 @@
     @Column(name = "check_out")
     private Time checkOut;
 
+    @NotBlank(message = "Please enter the hotel logo")
+    @Column(name = "image_logo")
+    private String imageLogo;
+
```
Comment (BojanJakic) [inline con-base/src/main/java/ba/codecentric/base/domain/Hotel.java]: You need additional annotation - max length of imageLogo is 128 characters

File: con-master/src/main/java/ba/codecentric/master/controller/HotelController.java [hunk 2]
```diff
@@ -2,31 +2,56 @@
...
+import org.springframework.web.multipart.MultipartFile;
 
 import javax.validation.Valid;
+import java.io.IOException;
 
 @RestController
 public class HotelController {
 
     private final HotelService hotelService;
-    private final static Logger LOG = Logger.getLogger(HotelController.class);
```
Comment (ozegster) [inline con-master/src/main/java/ba/codecentric/master/controller/HotelController.java]: Why did you remove Logger? You should log some events, eg. if hotel is properly saved in db, if image doesn't exist, if image is found and on which path.

File: con-base/src/main/java/ba/codecentric/base/repository/HotelRepository.java [hunk 3]
```diff
@@ -2,9 +2,16 @@
 
 import ba.codecentric.base.domain.Hotel;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
 
 public interface HotelRepository extends JpaRepository<Hotel, Integer> {
 
     Hotel findFirstByOrderById();
 
+    @Query("SELECT hotel.imageLogo FROM Hotel hotel WHERE hotel.id = ?1")
```
Comment (ozegster) [inline con-base/src/main/java/ba/codecentric/base/repository/HotelRepository.java]: Why do we need this query?

**Comment by ozegster** [comment]
image
Align rating and stars in the middle. Also, it is not nice to stretch placeholder image like this. And you can increase height of logo, but take care that same proportions are used in cropper.
